### ISSUE TITLE
Needs Error Logging

### ISSUE THREAD

**Issue by subtlecaffeine** [HeeminTV/NSOVVG#2]
It would be nice to know why the thing fails other than just going back to the menu. Does it not like my WAV files? Is there something in ffmpeg I'm missing I don't know I need?

**Comment by HeeminTV** [comment]
Hi Jay Moore, As you know, NSOVVG is a small project that is made because I
was bored.

`chcp 949` was written for test, and I forgot to remove the line. Actually
it worked for me and without it, some characters looked weirdly.

Also, making error handler is a bit difficult for me. The script just moves
user back to the menu screen right after the ffmpeg is closed(done), and I
have no idea how to make it show error messages if ffmpeg get error (I'll
test for %ERRORLEVEL% soon)

it's know what files you're working on and where you're having issues with
via email, so we can chat on discord if you prefer to communicate better.
(just asking)

the crash in ffmpeg when I used this program was as follows

- Has too many channels (ffmpeg command was too long)
- Invalid file format (showwaves filter sometimes fails to detect specific
file formats)
- ffmpeg, ffprobe, ffplay (if you want to see "preview") must be installed
in the same directory of the batch script or system path of your pc
(usually C:\Windows\System32 )

Sorry for not being of much help

Thank you for feedback!

(oops sorry I thought it was sent from gmail)

**Comment by subtlecaffeine** [comment]
I also apologize as I do understand this is a small for fun project. I got a little passionate and stressed.

I spent some time reading through the bat, remembering the art of scripting, and then added -report to the options and let her go. And nothing. I was able to verify the command line had been parsed, but the log just stopped with no explanation.

I downloaded a different build and dropped the exes in the folder I was working in. Now it works. That is an ffmpeg mystery I will solve later.

I'm impressed with this. I've done some bat scripting in my days and this is pretty complex. The progress screen did not look right, worked, but looked wrong. I think this is just code page/character encoding issues. When I get those figured out I will submit a pull request if you want to use it.
![1u2gdv00](https://github.com/user-attachments/assets/091734e5-b4a7-4d95-b7f2-ef1af1b779d8)
